scraper/spiders/meeting_spider.py

```diff
@@ -155,7 +155,7 @@ def get_meeting_detail(self,response):
 
         if "Women" in meeting_data['detail']:
             types.append("W") #  Women's meeting
-        elif "Men" in meeting_data['detail']:
+        elif "Men's" in meeting_data['detail'] or "Mens" in meeting_data['detail']:
```

Comment:
Does  `elif "Men" in meeting_data['detail']:` not cover both matches?
If not should we add for women as well?

Reply:
The problem is that `elif "Men" in meeting_data['detail']:` covers both matches AND strings like:

```
Mental hospital
```

I saw a handful of meetings incorrectly categorised because of this. This isn't a problem with the string `Women`.